### PR TITLE
Add numParallelCollections and viewsAsCollections options

### LINKED ISSUE #52: numParallelCollections and viewsAsCollections options are not available

### PR DESCRIPTION
**Pull request by chdanielmueller**
per https://www.mongodb.com/docs/v4.2/reference/program/mongodump/#mongodump-options
closes #52

### CODE REVIEW

**Review by boly38 — CHANGES_REQUESTED** [review]
welcome @chdanielmueller and thanks for your contribution 👍 

could you 
- put 2 more row in [readme mongodump options](https://github.com/boly38/node-mongotools#mongodump-options)
- create an issue and refer to it in your single commit msg please 🙏 (in fact the automated release process will include issues only to create release [notes](https://github.com/boly38/node-mongotools/releases))

**Comment by chdanielmueller** [comment]
Options are now documented and issue is open and referenced in the PR and commit message

File: README.md [hunk 1]
```diff
@@ -166,17 +166,19 @@ Optional ssl related options
...
+| `secret`                 | MT_SECRET | false    | null                    | secret to use if encrypt is enabled                                   |
+| `encryptSuffix`          |           | false    | `.enc`                  | encrypt file suffix                                                   |
+| `includeCollections`     |           | false    | (none)                  | **Deprecated** - please use `collection`                              |
+| `collection`             |           | false    | (none)                  | Collection to include, if not specified all collections are included  |
+| `excludeCollections`     |           | false    | (none)                  | Collections to exclude, if not specified all collections are included |
+| `numParallelCollections` |           | false    | 4                       | Number of collections mongodump should export in parallel.            |
+| `excludeCollections`     |           | false    | false                   | When specified, mongodump exports read-only views as collections.     |
```
Comment (boly38) [inline README.md:181]: `viewsAsCollections`

**Review by boly38 — APPROVED** [review]
LGTM
squash is recommended but not enforced.

with ou without squash I ll merge it in about 2h

**Comment by chdanielmueller** [comment]
You can do a squash merge if you want.
Happy you found my contribution worth to merge.
Thanks for taking the time to look at it

**Review by boly38 — APPROVED** [review]

**Pull request merged**